app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
we talked in class about how to leverage your associations here rather than setting the foreign key manually. how could you refactor this?